packages/trader/src/AppV2/Components/PurchaseButton/purchase-button.tsx

```diff
@@ -61,9 +73,9 @@ const PurchaseButton = observer(() => {
         is_vanilla_fx,
         is_vanilla,
     };
-    const trade_types_array = Object.keys(trade_types).filter(
-        type => !getTradeTypeTabsList(contract_type).length || type === trade_type_tab
-    );
+    const trade_types_array = Object.keys(trade_types)
+        .filter(type => !getTradeTypeTabsList(contract_type).length || type === trade_type_tab)
+        .sort((a, b) => Number(getSortedIndex(a) ?? 0) - Number(getSortedIndex(b) ?? 0));
```

Comment:
Which issue it's solving? + 'top' and 'bottom' are not applicable to current design as it's 'left' or 'right'

Reply:
solves Even/Odd buttons being swapped. getSortedIndex() is used in production to keep Even/Odd buttons in place